op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -29,7 +29,6 @@ func Test_ProgramAction_OperatorFeeConstistency(gt *testing.T) {
 
 	const testOperatorFeeScalar = uint32(20000)
```

Comment:
```suggestion
	const testOperatorFeeScalar = uint32(100e6)
```
since this scalar value is divided by 1e6, let's bump it up to make the bug more apparent